charts/internal/control-plane/templates/auditlog-webhook-deployment.yaml

```diff
@@ -0,0 +1,124 @@
+# kind: Secret
+# metadata:
+#   name: splunk-audit-webhook-server
+#   namespace: kube-system
+# type: Opaque
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: splunk-audit-webhook
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: splunk-audit-webhook
+spec:
+  selector:
+    matchLabels:
+      k8s-app: splunk-audit-webhook
+  template:
+    metadata:
+      labels:
+        k8s-app: splunk-audit-webhook
+        app: splunk-audit-webhook
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+      annotations:
+          scheduler.alpha.kubernetes.io/critical-pod: ''
+        spec:
+      # it's better to disable the service links as the default args do not work properly otherwise (metal#64)
+      # is this still necessary? Not in authn deployment.
+      enableServiceLinks: False
+      containers:
+      - name: splunk-audit-webhook
+        image: {{ index .Values.images "auditlog-webhook" }}
+        imagePullPolicy: Always
+        restartPolicy: Always
+        env:
+        - name: SPLUNK_AUDIT_WEBHOOK_SERVER_URLS
+          value: {{ .Values.auditlogWebhook.splunk-hec-endpoint.url }}
+        - name: SPLUNK_AUDIT_WEBHOOK_TOKEN
+          value: {{ .Values.auditlogWebhook.splunk-hec-endpoint.token }}
```

Comment:
I think it is not possible in these templates to access variables with hyphens like this. Camel Case will save you some time debugging.

Reply:
Thanks!